Fix crash on dedicated servers, optimise

PaintNinja opened the pull request:
Fixes #46.

Significantly optimised ClientTickHandler, especially the lowestFps/highestFps calculations.

PaintNinja committed:
Fix crash on dedicated servers, optimise

https://github.com/dimadencep/sodium-extra-forge/issues/46

Significantly optimised ClientTickHandler, especially the lowestFps/highestFps calculations

dima-dencep commented on src/main/java/me/flashyreese/mods/sodiumextra/SodiumExtraMod.java:
```diff
@@ -0,0 +1,9 @@
+package me.flashyreese.mods.sodiumextra;
+
+import net.minecraftforge.fml.common.Mod;
+
+@Mod(SodiumExtraMod.MOD_ID)
+public final class SodiumExtraMod {
+    public static final String MOD_ID = "sodiumextra";
+    public SodiumExtraMod() {}
+}
```
Why

PaintNinja replied:
Seriously? I did a whole write-up in the [associated issue](https://github.com/dimadencep/sodium-extra-forge/issues/46) explaining it.

dima-dencep replied:
Sorry, at first I only saw the pull request.

dima-dencep commented on src/main/resources/META-INF/mods.toml:
```diff
@@ -27,4 +27,11 @@ modId = "caffeineconfig"
\ No newline at end of file
+side = "CLIENT"
+
+[[dependencies.sodiumextra]]
+modId = "minecraft" # Gives a friendlier error message when installed on the wrong MC version
+mandatory = true
+versionRange = "[1.19.4,1.20)" # Allows 1.19.4 to 1.19.9
+ordering = "NONE"
+side = "CLIENT"
```
The original mod is independent of the minecraft version

PaintNinja replied:
If you run the 1.19.4 version of Rubidium Extras on 1.18.2, it crashes. This gives a more helpful error message saying something along the lines of "You have Minecraft 1.18.2 but (mod file name) requires 1.19.4".

The original mod will also crash if you attempt to use the wrong version for your MC.

PaintNinja committed:
Fix displaytest

Whoops, seems like Forge's displayTest mods.toml property is knackered atm, The JiJ'ed colamixinconfig is still causing the displaytest to fail however.

PaintNinja commented:
Test jars
[rubidium-extra-0.4.18+mc1.19.4-fix-crash-on-dedicated-servers.zip](https://github.com/dimadencep/sodium-extra-forge/files/11511268/rubidium-extra-0.4.18%2Bmc1.19.4-fix-crash-on-dedicated-servers.zip)

`forge mods` command in console to show rubidium extras not crashing the dedicated server:
<img width="877" alt="image" src="https://github.com/dimadencep/sodium-extra-forge/assets/3158390/24d8baa5-04ad-4923-966a-31a42517de08">
